activerecord/lib/active_record/callbacks.rb

```diff
@@ -234,30 +234,13 @@ module ActiveRecord
   #   end
   #
   # In this case the +log_children+ is executed before +do_something_else+.
-  # The same applies to all non-transactional callbacks.
+  # This applies to all non-transactional callbacks, and to `before_commit`.
   #
-  # As seen below, in case there are multiple transactional callbacks the order
-  # is reversed.
-  #
-  # For example:
-  #
-  #   class Topic < ActiveRecord::Base
-  #     has_many :children
-  #
-  #     after_commit :log_children
-  #     after_commit :do_something_else
-  #
-  #     private
-  #       def log_children
-  #         # Child processing
-  #       end
-  #
-  #       def do_something_else
-  #         # Something else
-  #       end
-  #   end
-  #
-  # In this case the +do_something_else+ is executed before +log_children+.
+  # For transactional `after_` callbacks (`after_commit`, `after_rollback`, etc), the order
+  # can be set by `config.active_record.run_after_transaction_callbacks_in_order_defined`.
+  # If `true` (the default from Rails 7.1), callbacks are executed in the order they
+  # are defined, just like the example above. If `false`, the order is reversed, so
+  # +do_something_else+ is executed before +log_children+.
```

Comment:
Maybe let's use consistent forms of backticks vs. plus for code escaping while we are here, I know this happens everywhere but why not start now?

Reply:
Wow yeah that's slack on my part. Will fix.

Reply:
@ghiculescu Thanks for making that change!

I figured you had just cut the text that was there which uses `+` and typed backticks out of habit :joy:

FWIW, I made some small suggestion here: ghiculescu/rails#2